### ISSUE TITLE
Bent long bone

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9613]
**Preferred term label:**
Bent long bone

**Synonyms**

**Definition (free text, please give PubMed ID)**
A developmental defect of a long bone, such as the femur or the humerus, characterized by a sharply curve deviation located in the metaphysis (shaft) of the bone.
PMID:25250905

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
614592 
PMID: 22387015  4/4

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
